deploy/helm/events-collector-config.yaml

```diff
@@ -116,6 +116,17 @@ processors:
         statements:
           # convert body to string
           - set(body, body.string)
+  transform/set_status_for_pods:
+    error_mode: ignore
+    log_statements:
+      - context: log
+        statements:
+          # status
+          - set(attributes["sw.k8s.pod.status"], body["status"]["phase"]) where body["kind"] == "Pod" and body["status"]["phase"] != nil
```

Comment:
This creates the attribute on the log level. We need it to be on the resource level - either put it in resource attributes explicitly or group by it to get it there implicitly.